app/controllers/wpcc/your_contributions_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
         session[:salary].to_i, session[:contribution_preference]
       )
 
-      store_eligible_salary
+      add_eligible_salary
```

Comment:
should be line above `store_eligible_salary`

kind of surprised the tests passed?